### DISCUSSION TITLE
FluentDataGrid and re-order rows via drag and drop ?

### DISCUSSION THREAD

**Discussion by bao86** [microsoft/fluentui-blazor#1023] [Q&A]
Hi there,

Is there a method for reordering rows in grid via drag and drop like this 
https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/draganddrop ?

Thanks in advance

**Comment by vnbaaij** [comment]
No, there is no such functionality in place (and we are not planning on adding that)
